community/lexicon/schema/getSchemas.json

```diff
@@ -0,0 +1,48 @@
+{
+  "lexicon": 1,
+  "id": "community.lexicon.schema.getSchemas",
+  "defs": {
+    "main": {
+      "type": "procedure",
```

Comment:
Should this be a `query` + `parameters` since it would be a non-modifying GET operation?

<img width="468" alt="Screenshot 2025-03-09 at 6 05 51 PM" src="https://github.com/user-attachments/assets/03cd3731-e01d-4c8e-8709-0a69b5cff2d8" />

Reply:
I meant to leave a comment about this. It's not really possible to express the input as parameters without inventing some new string format that's like `at://identity/nsid/rkey#cid` or something.

I'd prefer not to do that invention here (it should be an atproto core spec) but also stuffing those kinds of strings into the query string would reduce the amount of lexicons this endpoint could return I think (because of URL length limits of many systems and tooling).

Reply:
I'm not sure there needs to be any inventing here. Lexicon retrieval is more or less a `getRecord` which includes all 4 fields in the parameters

https://github.com/bluesky-social/atproto/blob/main/lexicons/com/atproto/repo/getRecord.json

Reply:
This endpoint returns multiple lexicons, not a single one. It's closer in spirit to app.bsky.feed.getPosts than it is to getRecord.

Reply:
There's literally no way to encode a "list of at URIs with optional CIDs" without inventing a new string format.